client/blocks/followers-count/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Count from 'components/count';
+import { getSelectedSite } from 'state/ui/selectors';
+
+class FollowersCount extends Component {
+	render() {
+		const { site, translate } = this.props;
+
+		if ( ! site || ! site.subscribers_count ) {
+			return null;
+		}
+
+		return (
+			<div className="followers-count">
+				<Button borderless href={ '/people/followers/' + site.slug }>
```

Comment:
This is not actually correct. `site.slug` is the right way to handle the site fragment for all sites—including those with subdirectories. `slugToUrl` was removed shortly after in https://github.com/Automattic/wp-calypso/pull/9338